lib/tasks/migration.rake

```diff
@@ -616,6 +621,12 @@ namespace :migration do
 
       MigrationLogger.info "Generic File saved id:#{@generic_file.id}"
       MigrationLogger.info "Generic File created id:#{@generic_file.id}"
+      puts "Updating handle #{fedora3handle} for migrated file #{@generic_file.id}"
+      MigrationLogger.info "Updating handle #{fedora3handle} for migrated file #{@generic_file.id}"
+      #This is to call a command line script to update the handle to redirect to ERA/noid
+      #Please follow the instruction here https://github.com/ualbertalib/era/tree/handle/handle to configure the scripts to work with the migration
+ 
+      update_handle(fedora3handle, @generic_file.id) if ENV['RAILS_ENV'] == 'production'
```

Comment:
Suggestions for testing in another environment? Not sure if I feel comfortable trusting that this will just work.